client/blocks/reader-post-card/index.jsx

```diff
@@ -88,6 +88,11 @@ class ReaderPostCard extends Component {
 			return;
 		}
 
+		// ignore clicks on comments
+		if ( closest( event.target, '.conversations__comment-list', rootNode ) ) {
+			return;
+		}
```

Comment:
Oh very nice! Il have to adopt this type of approach for the inline-comments as well.